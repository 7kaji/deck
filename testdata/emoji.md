# Title

---

# Title

## Subtitle

- 👉[here](https://example.com)
- here[👉](https://example.com)
- ➡️[here](https://example.com)
- 👍🏼[here](https://example.com)
