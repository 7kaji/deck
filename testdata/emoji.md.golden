[
  {
    "layout": "",
    "titles": [
      "Title"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Title"
    ],
    "subtitles": [
      "Subtitle"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "üëâ"
              },
              {
                "value": "here",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "here"
              },
              {
                "value": "üëâ",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "‚û°Ô∏è"
              },
              {
                "value": "here",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "üëçüèº"
              },
              {
                "value": "here",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]